src/plugins/expression/expression-types.ts

```diff
@@ -1,3 +1,39 @@
 export const kExpressionTileType = "Expression";
-
 export const kExpressionDefaultHeight = 100;
+
+export type SelectStatus = "empty" | "all" | "some" | "cursor" | undefined;
+export type InsertModeString = "replaceAll" | "insertAfter" | "replaceSelection";
+
+const ph = "\\placeholder{}";
+const emptyFrac = `\\frac{${ph}}{${ph}}}`;
+const mixedFrac = `${ph}\\frac{${ph}}{${ph}}`;
+const divSign = "\\div";
+const multSign = "\\times";
+const divisionEmpty = `${ph}${divSign}${ph}`;
+const multEmpty = `${ph}${multSign}${ph}`;
+
+export const latexStrings = {
+  ph,
+  emptyFrac,
+  mixedFrac,
+  divSign,
+  divisionEmpty,
+  multSign,
+  multEmpty
+};
+
+// Note: #@ token replaced by any highlighted value, otherwise placeholder
+export const expressionButtonsList = [
+  {
+    name: "divisionSymbol",
+    title: "Division Symbol",
+    className: "division-symbol",
+    baseLatex: `$#@${divSign}${ph}`
```

Comment:
What if the division button is pressed when the cursor is before number. Could you add a second `$#@` to the end of the Latex so it would only add the placeholder at the end if there is no number already after the division sign?

Reply:
I tried that but on its own it adds a second copy of the selection.  So if the value is 123 and the cursor is at the end, we end up with `123 ÷ 123`.   

But, it did work if I can adjust the LaTeX to put it on one side or the other.  I think I came up with something simple enough open the can of worms just enough to cover cases of "cursor blinking at the front of the expression".  

The `expressionButtonList` item becomes:

```ts
  {
    name: "divisionSymbol",
    title: "Division Symbol",
    className: "division-symbol",
    baseLatex: divSign,
    isBinaryOperator: true
  },
```

and then just before we pass the insert string:

```ts
  // if a binary operator, we need to add placeholders now
  // This may be replaced by generic solution based on computation: PT#185337722
  if (button?.isBinaryOperator){
    const atStart = selectStatus === "cursor" && mf.position === 0;
    insertString = atStart ?
      `\\placeholder{}${insertString}`
      : `#@${insertString}\\placeholder{}`;
  }

```

I am guessing this will be made irrelevant by the upcoming placeholder work, and/or that it has actually left the can of worms open, but let me know what you think.